renderer/reducers/channels.js

```diff
@@ -278,7 +278,11 @@ export const fetchChannels = () => async dispatch => {
 }
 
 // Receive IPC event for channels
-export const receiveChannels = ({ channels, pendingChannels, closedChannels }) => dispatch => {
+export const receiveChannels = ({
+  channels: { channels },
+  pendingChannels,
+  closedChannels: { channels: closedChannels },
```

Comment:
Are we sure that `channels` and `closedChannels`  and `closedChannels.channels` are always set?

Reply:
We are as much as the fact that we are calling lnd service methods directly and we know that those methods output data in this format.

Probably we can abstract this away but I think that's the job of some future refinements and should be done app-wide.